app/api/dao/user.py

```diff
@@ -130,8 +133,14 @@ def update_user_profile(user_id, data):
         if 'resume_url' in data and data['resume_url']:
             user.resume_url = data['resume_url']
 
-        if 'photo_url' in data and data['photo_url']:
-            user.photo_url = data['photo_url']
+        try:
+            if 'photo_url' in data and data['photo_url']:
+                image_type = data['photo_url'].split(';')[0].split('/')[1]
+                image_name = ".".join((username, image_type))
+                photo_url = aws_upload.upload_to_aws(data['photo_url'], os.environ.get("AWS_PROFILE_BUCKET"), image_name)
```

Comment:
Can you import this environment variable on [develop/config.py](https://github.com/systers/mentorship-backend/blob/develop/config.py) file, so that we use it here like the rest of the environments... `àpplication.config.ENV_VAR`

Reply:
I'll update this.